replica/database.hh

```diff
@@ -1092,6 +1093,7 @@ public:
     friend std::ostream& operator<<(std::ostream& out, const column_family& cf);
     // Testing purposes.
     friend class ::column_family_test;
+    friend class ::table_for_tests;
```

Comment:
Why this friendship? I only see public methods used below.

Reply:
AFAIK https://github.com/scylladb/scylladb/pull/13073 removes this so, when we merge that we should remember to remove this friendship.

Reply:
adding comment in next version